cmd/deployer/main.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform"
+	"github.com/mattermost/mattermost-load-test-ng/logger"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+
+	"github.com/spf13/cobra"
+)
+
+func RunCreateCmdF(cmd *cobra.Command, args []string) error {
+	config, err := getConfig(cmd)
+	if err != nil {
+		mlog.Error(err.Error())
+		return nil
+	}
+
+	t := terraform.New(config)
+	err = t.Create()
+	if err != nil {
+		mlog.Error(err.Error())
+	}
+	return nil
+}
+
+func RunDestroyCmdF(cmd *cobra.Command, args []string) error {
+	config, err := getConfig(cmd)
+	if err != nil {
+		mlog.Error(err.Error())
+		return nil
+	}
+
+	t := terraform.New(config)
+	err = t.Destroy()
+	if err != nil {
+		mlog.Error(err.Error())
+	}
+	return nil
```

Comment:
I am not sure if this is the best way to error. We are having an error but our executable will return 0 which seems successful. I think we should return error and print the recent error just before `os.Exit(1)`

Reply:
Yeah .. so I had mentioned it before in a PR. The issue is with how cobra handles the error. If I return the error from the command handler, then cobra thinks that it's an issue with the syntax and just prints the entire command help again, which is pretty annoying to me. Hence, we are printing the error and returning nil.

Yes, not the best way, but certainly better than this:

```
Error: failed to read config: unable to read configuration file: While parsing config: invalid character '\n' in string literal
Usage:
  deployer create [flags]

Flags:
  -h, --help   help for create

Global Flags:
  -c, --config string   path to the configuration file to use

error: failed to read config: unable to read configuration file: While parsing config: invalid character '\n' in string literal
exit status 1
```

Reply:
Hmm can we use this [flag](https://github.com/spf13/cobra/blob/0da06874266c88228b8f14615396a1f6bfc90ed7/command.go#L122)?

```golang
// SilenceUsage is an option to silence usage when an error occurs.
SilenceUsage bool
```

Reply:
Amazing .. that's perfect ! I will do it in a separate PR if you don't mind.